### PR TITLE
fix(list): remove default connotation

### PR DESCRIPTION
**Pull request by yinonov**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by rachelbt** [comment]
the activated state is broken:
image

**Comment by yinonov** [comment]
> the activated state is broken: image was: image

fixed

**Comment by rachelbt** [comment]
So now we do not support cta connotation in list-item?
https://tmp.dev.vivid.vonage.com/1256/index.html?path=/story/components-list-list-item--connotation&args=connotation:primary;activated:;shape:rounded;graphic:icon

**Comment by yinonov** [comment]
> So now we do not support cta connotation in list-item? https://tmp.dev.vivid.vonage.com/1256/index.html?path=/story/components-list-list-item--connotation&args=connotation:primary;activated:;shape:rounded;graphic:icon isn't it kind of a breaking design change?

according to @jshenkman it's actually a fix as we shouldn't have supported it

**Comment by jshenkman** [comment]
@yinonov @rachelbt 
exactly, no product uses CTA connotation is list or side navigation

**Comment by rachelbt** [comment]
something looks a bit off with the icon that is not white:
image

also when pressed the ripple is hiding the icon

**Comment by yinonov** [comment]
> here

@jshenkman mind reviewing if this is tolerable? it's an issue with the side-navigation variant, which will deprecate following the new sidenav

**Comment by jshenkman** [comment]
Ok, so a few problems - 
1. `list item - activated` - the activated item shouldn't be bold, the text style should stay the same.
2. `list item - connotation` the list item is a bit different, it's all black (text + icon) and on selection, it turns white. only on item can be selected and the last one is supposed to be disabled so it shouldn't be activated.

Screen Shot 2022-03-20 at 10 48 28

I'm not sure what you would want to fix now and what to leave as is and fix in vivid-3, but in my opinion, the list needs to be prioritized over the side nav if they collide.

File: components/list/src/vwc-list-item.scss [hunk 1]
```diff
@@ -114,3 +128,16 @@ $vvd-list-item-graphic-margin: --vvd-list-item-graphic-margin;
 .mdc-deprecated-list-item__meta {
 	display: flex;
 }
+
+
+:host(:not([disabled])) {
```
Comment (rachelbt) [inline components/list/src/vwc-list-item.scss:133]: this causes the icon to be white when its not active or disabled:
image

**Review by rachelbt — COMMENTED** [review]
sorry for being a nag but when there are 2 lines option & connotation primary + active the bottom line is unreadable:
image

**Comment by yinonov** [comment]
@jshenkman mind reviewing @rachelbt comments and guide on design?

**Comment by jshenkman** [comment]
> sorry for being a nag but when there are 2 lines option & connotation primary + active the bottom line is unreadable: image

This option should be able to get the primary connotation as it's not a side-nav list item. 
So can we block this? or remove this property from all list items? 

Active connotation is only for side-nav list item

**Comment by yinonov** [comment]
> > sorry for being a nag but when there are 2 lines option & connotation primary + active the bottom line is unreadable: image
> 
> 
> 
> This option shouldn't be able to get the primary connotation as it's not a side-nav list item. 
> 
> So can we block this? or remove this property from all list items? 
> 
> 
> 
> Active connotation is only for side-nav list item 

Not really, the only indication list-item is used as sidenav-item is it is set with connotation. We haven't added  any explicit interface. Tech debt we bought into.
Can hide secondary line for items with connotation but this can result in a very confusing interface

**Comment by jshenkman** [comment]
> > > sorry for being a nag but when there are 2 lines option & connotation primary + active the bottom line is unreadable: image
> > 
> > 
> > This option shouldn't be able to get the primary connotation as it's not a side-nav list item.
> > So can we block this? or remove this property from all list items?
> > Active connotation is only for side-nav list item
> 
> Not really, the only indication list-item is used as sidenav-item is it is set with connotation. We haven't added any explicit interface. Tech debt we bought into. Can hide secondary line for items with connotation but this can result in a very confusing interface

ok got it, so the list item should act like in dark theme, the secondary line should switch from neutral-70 to neutral-40

**Review by rachelbt — APPROVED** [review]
Now its perfect :)

**Pull request merged**